CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Added
 
-*
+* Fedora 29+ is know supported by certbot-auto. Since Python 2.x starts to be
+  deprecated with Fedora 29, certbot-auto will install and use Python 3.x instead.
```

Comment:
Is it true that Python 2 is deprecated in Fedora 29? They're definitely going that direction, but looking at https://fedoraproject.org/wiki/FinalizingFedoraSwitchtoPython3, it seems to me like that's coming later.

If that's the case or you're not sure, I'd make this more generic and say something like:

> Since Python 2.x is on a deprecation path in Fedora, certbot-auto will install and use Python 3.x on Fedora 29+.

Reply:
Yes you are right, I am a little too much assertive here.